PySpice/Spice/HSpice/hspicefile/hspice_read.c

```
@@ -0,0 +1,972 @@
+				  int *rawDataOffset)
+{
+	int error, blockHeader[blockHeaderSize], swap;
+	float *tmpRawData;
+
+	if (debugMode>=2) {
+		fprintf(debugFile, "Reading data block @0x%lx\n", ftell(f));
+	}
+
+	// Get size of raw data block.
+	swap = readBlockHeader(f, fileName, debugMode, blockHeader, sizeof(float));
+	if(swap == -2) return -1; // Normal EOF / End of block.
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)